package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "./node_modules/.bin/mocha ./tests/*test.js"
```

Comment:
good

you can also add ./tests/**.*.test.js in case you want pick up tests in any sub directory if you started structuring your test cases